projects/packages/assets/changelog/add-script-strategy-support

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Possibly we should make this major, since it's dropping support for WP before 6.3.